lib/task/import/csvImportTask.class.php

```diff
@@ -144,6 +151,16 @@ public function execute($arguments = array(), $options = array())
       throw new sfException('You must specify a valid filename');
     }
 
+    if (!empty($options['user-id']))
+    {
+      $criteria = new Criteria;
+      $criteria->add(QubitUser::ID, $options['user-id']);
+       if (null != $user = QubitUser::getOne($criteria))
```

Comment:
Extra whitespace and, now that you're in there, I think you could use directly `getById`.